testing/factory/factory.go

```diff
@@ -90,6 +90,7 @@ type MachineParams struct {
 	Nonce           string
 	Constraints     constraints.Value
 	InstanceId      instance.Id
+	DisplayName     string
```

Comment:
Where is this used? Looks like still need to add some code?

Reply:
Have added the DisplayName to default settings within the factory, but haven't modified any tests to make use of the field directly yet.